AngularApp/src/app/shared/employee.service.ts

```diff
@@ -1,8 +1,16 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+//changes for latest rxjs imports as previous were not working
+/*
 import { Observable } from 'rxjs/Observable';
+
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/toPromise';
+*/
+import { Observable } from 'rxjs';
+
+import { map } from 'rxjs/operators';
+import { toPromise } from 'rxjs/operators';
 
 import { Employee } from './employee.model';
 
```

Comment:
In the latest version of rxjs, we dont need to import toPromise.js since it is included in the Observable itself.

Reply:
still not working properly. when I insert this "import { toPromise } from 'rxjs/operators';" , toPromise has been underlined and    
without these line " import { Observable } from 'rxjs';

import { map } from 'rxjs/operators';
import { toPromise } from 'rxjs/operators';"

or within these lines cannot load the web page.

Reply:
still not working properly. when I insert this "import { toPromise } from 'rxjs/operators';" , toPromise has been underlined and
without these line " import { Observable } from 'rxjs';

import { map } from 'rxjs/operators';
import { toPromise } from 'rxjs/operators';"

or within these lines cannot load the web page.

Reply:
I follow your tutorial.but my submit button is not working.and when i click the submit button the data is not displaced please help me to solve this issue.